plugins/issue_prefix/__init__.py

```diff
@@ -7,7 +7,9 @@ def build_filter(args):
 
 class Filter:
     def __init__(self, args):
-        self.prefix = args.encode('utf8')
+        if not isinstance(args, bytes):
+            args = args.encode('utf8') 
+        self.prefix = args
```

Comment:
Please squash this into the previous commit as it has not gotten into master yet and as I had a suggestion for the readme.